content/blog/pulumi-release-notes-106/index.md

```diff
@@ -0,0 +1,139 @@
+
+### New Project Wizard Enhancements
+
+We enhanced the [New Project Wizard](/docs/pulumi-cloud/developer-portals/new-project-wizard/) in Pulumi Cloud to streamline developer workflows and give platform teams more control over resource deployments.  You can now configure the [Developer Portal Gallery](/blog/developer-portal-gallery/) to exclusively display organization-specific templates, ensuring developers only use approved configurations. This simplifies template discovery and selection, reinforcing governance and compliance across development projects. We also streamlined project creation by introducing default repository locations and auto-generated project names, eliminating manual steps and potential naming conflicts. Read the [blog post](/blog/new-project-wizard-apr2024-updates/) for more details.
+
+## Core
+
+### Continue On Error in the Pulumi CLI
+
+Pulumi now offers more control over error handling during deployments with the new `--continue-on-error` flag for `pulumi up` and `pulumi destroy`. When managing complex infrastructure with Pulumi, deployments can involve a large number of concurrent resource operations. Previously, if an error occurred during a deployment, Pulumi would stop all new operations to prevent potential cascading failures. While this approach is often desirable, there are cases where you might want to continue updating or destroying resources that are independent of the failed resource. The --continue-on-error flag enables this behavior, allowing deployments to proceed even in the presence of errors, while still respecting resource dependencies. This can be especially useful when destroying resources, as it allows you to clean up as much infrastructure as possible. Read the [blog post](/blog/continue-on-error/) for more details.
+
+{{< video title="Using pulumi new with the new AI option" src="/blog/continue-on-error/coe.mp4" controls="false" autoplay="true" loop="true" >}}
+
+### Run Pulumi with Any TypeScript Version
+Previously, Pulumi's seamless TypeScript experience was limited to version 3.8, requiring manual build steps for newer versions. But now, with the release of the [Pulumi Node.js SDK 3.113.0](https://www.npmjs.com/package/@pulumi/pulumi), you can use any TypeScript version 3.8 or later, including the latest [TypeScript 5.4](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-4.html), by simply adding it as a dependency in your project's package.json file. Pulumi will automatically use the specified version for compilation, unlocking access to the newest TypeScript features and improved type checking. For backward compatibility and ease of use within monorepos, TypeScript and ts-node are optional peer dependencies and Pulumi defaults to bundled versions of TypeScript 3.8 and ts-node 7 if no specific versions are declared. Read the [blog post](/blog/typescript-versions/) for a detailed example and instructions on upgrading your projects.
```

Comment:
I'd just cut this sentence:

```suggestion
Previously, Pulumi's seamless TypeScript experience was limited to version 3.8, requiring manual build steps for newer versions. But now, with the release of the [Pulumi Node.js SDK 3.113.0](https://www.npmjs.com/package/@pulumi/pulumi), you can use any TypeScript version 3.8 or later, including the latest [TypeScript 5.4](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-4.html), by simply adding it as a dependency in your project's package.json file. Pulumi will automatically use the specified version for compilation, unlocking access to the newest TypeScript features and improved type checking. Read the [blog post](/blog/typescript-versions/) for a detailed example and instructions on upgrading your projects.
```

Alternatively, say something like. "New Pulumi TypeScript projects depend on the latest version of TypeScript by default.".